### PR TITLE
doc: submodule doc change

### PR DESCRIPTION
**Pull request by srivickynesh**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce. Please also list any relevant details
for your test configuration

# Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added meaningful description with JIRA/GitHub issue key (if applicable)
- [ ] I have updated labels (if needed)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ Kubernetes Without Kubelet. Kwok simply simulates the node's behaviour.
 As a result, it can mimic a high number of nodes and pods while consuming
 only a small amount of memory.
 
+Run the following cmd for a fresh clone to initialize and update the submodule:
```
Comment (yftacherzog) [inline CONTRIBUTING.md:20]:
```suggestion
Run the following command for a fresh clone to initialize and update the submodule:
```

**Review by ifireball — CHANGES_REQUESTED** [review]
Please make the fix @yftacherzog suggested and I'll merge this.

**Review by ifireball — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -17,6 +17,11 @@ Kubernetes Without Kubelet. Kwok simply simulates the node's behaviour.
 As a result, it can mimic a high number of nodes and pods while consuming
 only a small amount of memory.
 
+Run the following command for a fresh clone to initialize and update the submodule:
```
Comment (Omeramsc) [inline CONTRIBUTING.md:20]: A similar instruction (or an anchor) should be added to the Unit Test section, as the tests will fail without running this command.

**Review by Omeramsc — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (srivickynesh) [reply CONTRIBUTING.md:20]: Done @Omeramsc

**Review by Omeramsc — APPROVED** [review]
LGTM

**Pull request merged**